ui/client/components/graph/node-modal/editors/expression/StringEditor.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
```

Comment:
I don't think we should have suggestions in "simple" mode? they won't work anyway?

Reply:
What about completion suggestions like: #meta.processName ? They still work.

Reply:
Change to standard input, without suggestions